nuitka/nodes/shapes/BuiltinTypeShapes.py

```diff
@@ -123,6 +123,10 @@ def getTypeName():
     def hasShapeSlotBool():
         return False
 
+    @staticmethod
```

Comment:
Unfortunately Github diff sucks here and doesn't show the classes, so I am not checking these at this time. Arguably, we need to create test cases for Nuitka that give coverage for these. I have made mistakes in this myself in released versions in the past.

Reply:
BTW: I node I can enable side by side diff, I am not doing now though.

Reply:
In a nutshell, I have set `True` only for `int`, `float` and `complex` related shape types.